examples/em_kraken.py

```diff
@@ -0,0 +1,99 @@
+# Builtin imports
```

Comment:
Tests the Kraken broker streamer functions and creates a candle graph of assets user selects.